kratos/tests/test_kratos_parameters.py

```diff
@@ -1,666 +0,0 @@
-﻿from __future__ import print_function, absolute_import, division

```

Comment:
this must not be deleted!

Reply:
Ok this was my fault, probably doing tests with the serialization I deleted it